resources/wikia/modules/cookiePolicy.js

```diff
@@ -0,0 +1,55 @@
+/* global GlobalNotification */
+require(['wikia.cookies', 'wikia.geo'], function (cookies, geo) {
+	'use strict';
+
+	/**
+	 * Initialize JS for whether or not to show a banner notifying European Union users that we use cookies
+	 * on our site, per EU law. Users can dismiss the notification and will not be shown again, unless
+	 * they clear their cookies.
+	 */
+	function initCookieNotification() {
+		if (shouldShowBanner()) {
+			showBanner();
+		}
+	}
+
+	/**
+	 * Check if user is from the EU and hasn't seen the banner yet
+	 * @returns {boolean}
+	 */
+	function shouldShowBanner() {
+		var fromEU = geo.getContinentCode() === 'EU';
+
+		return (fromEU && !hasSeenBanner());
+	}
+
+	/**
+	 * Display the cookie policy message to the user
+	 */
+	function showBanner() {
+		var message = $.msg('oasis-eu-cookie-policy-notification-message') +
```

Comment:
I know MW 1.19 doesn't support message parsing on frontend but messages concatenation is pain for translators, additionally Privacy Policy link is not allowed to be changed in different languages - there are languages that already translated this page: http://pl.wikia.com/Polityka_Prywatno%C5%9Bci

On the other hand, if I recall correctly PHP parser doesn't support ```target="_blank"```. 
Will investigate this topic more today.